docs_guide/intro/new_doc_project.rst

```diff
@@ -170,7 +170,7 @@ in ``docs/index.rst``.  Your ``index.rst`` should look like this:
        API Reference <apidocs/index>
        Explanations <explanations/index>
        Release Notes <release_notes>
-       GitHub <https://github.com/hal/qiskit-hal-proivder>
+       GitHub < <Enter Link to your Repository, e.g. https://github.com/hal/qiskit-hal-proivder> >
```

Comment:
```suggestion
       GitHub <Enter Link to your Repository, e.g. https://github.com/hal/qiskit-hal-proivder>
```

I think people might be confused if they should have two `<`s or not. The spaces should already make this clear you need to fill it in